base_tier_validation_definition_server_action/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,5 @@
+On each Tier Definition
+
+* Choose the **Server Action** option in the Definition.
+    * Server action should return True in order to add validation on object
```

Comment:
This is not correct looking at [tests](https://github.com/OCA/server-ux/pull/783/files#diff-a1a1aa2035bae8b54428317a1b58cc81bc0344b5ce732a0695bf208fae6c26d8R74) and my functional review. It looks like an action variable is expected, isn't it?

Reply:
Yes, it is

Reply:
Its updated,

Thanks